README/Case.agda

```diff
@@ -15,7 +15,7 @@ open import Data.Nat   hiding (pred)
 open import Data.List
 open import Data.Sum
 open import Data.Product
-open import Function
+open import Function -- case_of_ defined here
```

Comment:
Why not *explicitly* listing the things imported here?
And then if you want to, you can mention `Function`
again in running text closer to a use site.

Reply:
As a largely orthogonal issue, shouldn't this be `Function.Base` these days? Perhaps this would even make navigation easier.

Reply:
> As a largely orthogonal issue, shouldn't this be Function.Base these days? Perhaps this would even make navigation easier.

I guess it depends whether people want ease of use or to cut down on the number of dependencies...